### DISCUSSION TITLE
Schedule of releasing v1.6.0

### DISCUSSION THREAD

**Discussion by CairneRain** [loki4j/loki-logback-appender#275] [Q&A] [answered]
Hi,
Kindly ask when will v1.6.0 officially release? It will be nice to use structured metadata😀

Thank you

**Comment by nehaev** [comment, marked as answer]
Hi @CairneRain, thanks for you interest in Loki4j!

There is no specific schedule for v1.6.0. I plan to publish one more RC with some changes to `KeyValuePairsJsonProvider`, then wait for a few weeks and publish a release if nothing serious is reported.

If you are eager to try structured metadata feel free to use v1.6.0-rc1. I don't think this part will change until the release. You can find the most recent doc version [here](https://github.com/loki4j/loki-logback-appender/blob/docs-for-v1.6.0/docs/docus/docs/metadata.md).
